README.md

```diff
@@ -1,62 +1,62 @@
+ZCBOR can also validate and convert CBOR data to and from JSON/YAML, either from the command line, or imported as a module.
 Finally, the package contains a light-weight CBOR encoding/decoding library in C.
 This library is used by the generated code, and can also be used directly in your own code.
 
 Features
 ========
 
-Here are some possible ways cddl-gen can be used:
+Here are some possible ways ZCBOR can be used:
 
  - Python scripting:
    - Validate a YAML file and translate it into CBOR e.g. for transmission.
    - Validate a YAML/JSON/CBOR file before processing it with some other tool
    - Decode and validate incoming CBOR data into human-readable YAML/JSON.
-   - As part of a python script that processes YAML/JSON/CBOR files. cddl-gen is compatible with PyYAML and can additionally provide validation and/or easier inspection via named tuples.
+   - As part of a python script that processes YAML/JSON/CBOR files. ZCBOR is compatible with PyYAML and can additionally provide validation and/or easier inspection via named tuples.
  - C code:
    - Generate C code for validating and decoding or encoding CBOR, for use in optimized or constrained environments, such as microcontrollers.
    - Provide a low-footprint CBOR decoding/encoding library similar to TinyCBOR/QCBOR/NanoCBOR.
 
 Python scripting
 ================
 
-Invoking cddl_gen.py from the command line
-------------------------------------------
+Invoking zcbor.py from the command line
+---------------------------------------
 
-The cddl_gen.py script can directly read CBOR, YAML, or JSON data and validate it against a CDDL description.
+The zcbor.py script can directly read CBOR, YAML, or JSON data and validate it against a CDDL description.
 It can also freely convert the data between CBOR/YAML/JSON.
 It can also output the data to a C file formatted as a byte array.
 
 The following is a generalized example for converting (and validating) data from the command line.
 The script infers the data format from the file extension, but the format can also be specified explicitly.
-See `cddl-gen convert --help` for more information.
+See `ZCBOR convert --help` for more information.
 
 ```sh
-python3 <cddl-gen base>/cddl_gen/cddl_gen.py -c <CDDL description file> convert -t <which CDDL type to expect> -i <input data file> -o <output data file>
+python3 <ZCBOR base>/zcbor/zcbor.py -c <CDDL description file> convert -t <which CDDL type to expect> -i <input data file> -o <output data file>
 ```
 
 Or invoke its command line executable (if installed via `pip`):
 
 ```sh
-cddl-gen -c <CDDL description file> convert -t <which CDDL type to expect> -i <input data file> -o <output data file>
+zcbor -c <CDDL description file> convert -t <which CDDL type to expect> -i <input data file> -o <output data file>
```

Comment:
Here the name is lower case, so I guess we should change the places where it is upper case.